NDTensors/src/lib/TypeParameterAccessor/src/TypeParameterAccessor.jl

```diff
@@ -1,17 +1,12 @@
-module SetParameters
+module TypeParameterAccessor
```

Comment:
```suggestion
module TypeParameterAccessors
```
analogous to how `Accessors.jl` is plural.

Reply:
Okay i'll fix this across the board right now, thanks!